derive_builder/src/options/struct_mode.rs

```diff
@@ -1,15 +1,18 @@
 use syn;
-use options::{OptionsBuilder, OptionsBuilderMode, parse_lit_as_string, FieldMode, StructOptions};
+use options::{OptionsBuilder, OptionsBuilderMode, parse_lit_as_string, parse_lit_as_bool, parse_lit_as_path, FieldMode, StructOptions};
 use derive_builder_core::{DeprecationNotes, Bindings};
 
 #[derive(Debug, Clone)]
 pub struct StructMode {
+    build_fn_name: Option<String>,
+    build_fn_enabled: bool,
```

Comment:
It's probably possible - I'm emulating the existing setter code, which handled these attributes separately.